packages/nodes-base/nodes/If.node.json

```
@@ -2,6 +2,7 @@
 	"node": "n8n-nodes-base.if",
 	"nodeVersion": "1.0",
 	"codexVersion": "1.0",
+	"details": "The IF node can be used to implement binary conditional logic in your workflow. You can setup one to many conditions to evalute for each item of data being inputted into the node. That data will either evaluate to TRUE or FALSE and route out of the node accordingly. \n\nThis node has multiple types of conditions: Bool, String, Number, and Date & Time.",
```

typo and cleaning up text
```suggestion
	"details": "The IF node can be used to implement binary conditional logic in your workflow. You can setup one-to-many conditions to evaluate each item inputted into the node. That data will either evaluate to TRUE or FALSE and route out of the node accordingly. \n\nThis node has multiple types of conditions: Bool, String, Number, and Date & Time.",
```